### PR TITLE
Add Hand Tremor Module

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This is a modified version of the tremor module included in mPower v1.3 and builds upon #747 and #743. This version includes using the navigation rules and an ORKQuestionStep to skip a hand if the user indicates that they are not able to perform the activity with that hand.

### CODE REVIEW

**Comment by syoung-smallwisdom** [comment]
@YuanZhu-apple Thank you so much for all of your time over the past few weeks. This should be the last of the changes from our branch (for a while) that we have incorporated into our apps.

File: ResearchKit/Common/ORKInstructionStep.h [hunk 1]
```diff
@@ -63,6 +63,8 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic, copy, nullable) UIImage *image;
 
+@property (nonatomic, copy, nullable) UIImage *auxiliaryImage;
```
Comment (YuanZhu-apple) [inline ResearchKit/Common/ORKInstructionStep.h]: What's is this?

Comment (syoung-smallwisdom) [inline ResearchKit/Common/ORKInstructionStep.h]: This is a module developed by Thread that we are using in mPower. The auxiliaryImage is a second image used on some screens. The auxiliary image is overlaid with the main image to show a two-part image where only one part is tinted.

simulator screen shot aug 16 2016 1 20 55 pm

**Comment by YuanZhu-apple** [comment]
@syoung-smallwisdom 
Found one issue, there is an chunk of unused space above the title on one of the instruction screen.  
And can you provide a dedicated test in ORKTest to test the new feature: auxiliaryImage?

simulator screen shot aug 18 2016 3 50 06 pm

**Comment by syoung-smallwisdom** [comment]
@YuanZhu-apple 
Added example for just the aux image.

WRT the empty space above the text, this has not been changed with this module from the default layout defined in ResearchKit. Modifications to the layout to be "smarter" about calculating space is outside the bounds of this pull request.

File: ResearchKit/Common/ORKInstructionStep.h (hunk 1 above)
Comment (YuanZhu-apple) [inline ResearchKit/Common/ORKInstructionStep.h]: Can you document it?

Comment (syoung-smallwisdom) [inline ResearchKit/Common/ORKInstructionStep.h]: @YuanZhu-apple 
Done

**Comment by YuanZhu-apple** [comment]
@syoung-smallwisdom Thanks!

**Comment by YuanZhu-apple** [comment]
r=me!

**Pull request merged**